### PR TITLE
feat: add dataSetId support to template groups 

### PR DESCRIPTION
**Pull request by Tapped**

### CODE REVIEW

**Comment by andersfylling** [comment]
If you can attach the open api preview link and update the branch, then i can review this :+1:

File: packages/stable/src/types.ts [hunk 1]
```diff
@@ -2976,13 +2981,23 @@ export type TemplateGroup = ExternalTemplateGroup & {
    * When resource was last updated
    */
   lastUpdatedTime: Timestamp;
+
+  /**
+   * The dataSetId of the template group
+   */
+  dataSetId?: number;
```
Comment (peternic) [inline packages/stable/src/types.ts]: Does it not get this from the ExternalTemplateGroup?

**Review by mortenhillbom — APPROVED** [review]

File: packages/stable/src/types.ts (hunk 1 above)
Comment (Tapped) [reply packages/stable/src/types.ts]: Ah yes, it does, thank you!

**Comment by andersfylling** [comment]
your commit log is faulty, as indicated by the lint CI. @Tapped

**Comment by Tapped** [comment]
@andersfylling Thanks for letting me know!

**Review by andersfylling — APPROVED** [review]

**Pull request merged**